application.go

```diff
@@ -99,6 +99,7 @@ type Application struct {
 	LastTaskFailure       *LastTaskFailure        `json:"lastTaskFailure,omitempty"`
 	Fetch                 *[]Fetch                `json:"fetch,omitempty"`
 	IPAddressPerTask      *IPAddressPerTask       `json:"ipAddress,omitempty"`
+	Residency             *Residency              `json:"residence,omitempty"`
```

Comment:
Ah, thanks, i just switched to another task before be able to test it. Will fix it and do some testing to make sure it works as expected.